PSSlack/Public/New-SlackMessageAttachment.ps1

```diff
@@ -223,6 +261,15 @@ function New-SlackMessageAttachment
             $true
         })]
         [System.Collections.Hashtable[]]$Fields,
+
+        [Parameter(Mandatory=$true,
+                    ParameterSetName='Action')]

```

Comment:
@Rukas Is this parameter set needed? When specifying an action, you can't also specify `Color` or `Severity` because they're in different parameter sets.

Reply:
That's a good point. I used that as an easy way to make sure that a callback ID is specified when an action is specified. I guess I could add a `BEGIN` block and have it validate that both were specified and throw if one is but the other isn't.  Any thoughts on a better way?

Reply:
Good catch Brandon!

Yeah, just logic to validate that they're together, and throw if not might make the most sense?  Probably more readable / less complex than fun with multiple parametersets (e.g. color-action, color, severity-action, severity, or whatever combination we'd need to get them both in place), and should still be usable enough.  What do you all think?

Reply:
I've added the changed specified above. Let me know what you think!